docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -156,6 +158,7 @@ temporary directory by adding ``-Djava.io.tmpdir=/path/to/other/tmpdir`` to the
 list of JVM options.
 
 We enable ``-XX:+UnlockDiagnosticVMOptions`` and ``-XX:+UseAESCTRIntrinsics`` to improve AES performance for S3, etc. on ARM64 (`JDK-8271567 <https://bugs.openjdk.java.net/browse/JDK-8271567>`_)
+We disable prevent GC (``-XX:-G1UsePreventiveGC``) for performance reasons (see `JDK-8293861 <https://bugs.openjdk.org/browse/JDK-8293861>`_)
```

Comment:
nit: prevent -> preventive

Reply:
thanks @losipiuk for correcting my mistakes (https://github.com/trinodb/trino/pull/15380#discussion_r1048185956)